src/class/SimpleData.ts

```diff
@@ -131,26 +150,26 @@ export default class SimpleData {
 
     @logCall()
     summarize({
-        value,
-        keyCategories,
+        keyValue,
+        keyCategory,
         summary,
         weight,
         overwrite = false,
         nbDigits = 1,
         nbValuesTestedForTypeOf = 1000
     }: {
-        value?: string,
-        keyCategories?: string,
+        keyValue?: string | string[],
+        keyCategory?: string | string[] | undefined,
```

Comment:
Would the linter complain? I thought that `undefined` was implicit here since the parameter is marked as optional with `?`

Reply:
I think it complained. And that's why I added it. This little capricious linter!